includes/functions.php

```diff
@@ -64,7 +64,7 @@ function caldera_forms_fix_license_manger_link(){
  */
 function caldera_forms_woo_nonce_fix( $user_id, $action) {
 	if ( class_exists( 'WooCommerce' ) ) {
-		if ( $user_id && $user_id != 0 && $action && $action == 'caldera_forms_front' ) {
+		if ( $user_id && $user_id !== 0 && $action && strpos( $action, 'caldera_forms_front' ) !== false ) {
```

Comment:
@ictbeheer We use yoda conditionals. The old code is wrong in this manner.

`if ( $user_id &&  0 != $user_id && $action && 0 === strpos( $action, 'caldera_forms_front' )  )` would be more appropriate. Also, using `0 === strpos(`... ensures it is the start of the string `$action`